intro-to-ruby.rb

```diff
@@ -1,14 +1,20 @@
 #create a string and store it in a variable
+food = "chicken,french fries,pizza"
+puts " like to eat #{food} after school."
 
 #print out the value of the variable you just created
 
+
```

Comment:
you wrote the solution to this on line 3, which doesn't matter functionally but organizationally is difficult to read.